pkg/qemu/qemu.go

```diff
@@ -47,10 +47,43 @@ type Config struct {
 	VirtioGA     bool
 }
 
-// MinimumQemuVersion is the minimum supported QEMU version.
-const (
-	MinimumQemuVersion = "4.0.0"
-)
+// HardMinimumVersion is the hard minimum version of QEMU.
+// The driver immediately returns the error when QEMU is older than this version.
+func HardMinimumVersion() string {
```

Comment:
I don't think these functions need to be exported.

Reply:
fixed